components/locale_canonicalizer/tests/locale_canonicalizer.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+use icu_locale_canonicalizer::LocaleCanonicalizer;
+use icu_locid::Locale;
+use icu_locid_macros::langid;
+
+#[test]
+fn test_maximize() {
```

Comment:
What made you decide to put these tests in a separate file instead of at the bottom of the library file?

Reply:
I like them in separate file, but I'd like them in fixture even more ;) Then we could use it to test the functionality between ICU4C, ICU4J, and various outputs of ICU4X (and ECMA-402!)

Reply:
I'll break them out to a fixture.